spec/cases.json

```diff
@@ -3270,7 +3270,7 @@
         "hashVersion": 2,
         "variations": [0, 1],
         "weights": [0.5, 0.5],
-        "coverage": [0.99]
+        "coverage": 0.99
```

Comment:
Is this a legit fix or should the SDK be ready to receive also arrays here? 🤔

Reply:
Coverage should be a float. There was an error in the initial cases.json file where it was specified as an array of floats.